pkg/container/network.go

```diff
@@ -293,3 +488,33 @@ func maskString(mask net.IPMask) string {
 
 	return fmt.Sprintf("%d.%d.%d.%d", mask[0], mask[1], mask[2], mask[3])
 }
+
+// this function extracts a list of interfaces from VM's API definition
+// currently it's a comma-separated string stored in annotations
+func parseExtraIntfs(vm *api.VM) map[string]string {
+	result := make(map[string]string)
+
+	for k, v := range vm.GetObjectMeta().Annotations {
+		if !strings.HasPrefix(k, constants.IGNITE_INTERFACE_ANNOTATION) {
+			continue
+		}
+
+		k = strings.TrimPrefix(k, constants.IGNITE_INTERFACE_ANNOTATION)
+		if k == "" {
+			continue
+		}
+
+		switch v {
+		case supportedModes.dhcp:
+			result[k] = v
+		case supportedModes.tc:
+			result[k] = v
+		default:
+			continue
```

Comment:
Should we print a warning for unsupported mode here?
Ideally, it will show up in the ignite spawn log.